nucliadb_utils/nucliadb_utils/storages/pg.py

```diff
@@ -188,7 +204,16 @@ async def move(
                 origin_kb,
                 origin_key,
             )
-
+            # make sure to delete the destination first in
```

Comment:
Just out of curiosity: I don't understand why we need a delete here. Can't we just overwrite the existing values?

Reply:
This is essentially doing a replace.

The new upload is in a different file id and this clears out the old

Reply:
This is essentially doing a replace.

The new upload is in a different file id and this clears out the old